samples/cellular/nrf_cloud_multi_service/README.rst

```diff
@@ -557,20 +568,113 @@ CONFIG_AT_CMD_REQUEST_RESPONSE_BUFFER_LENGTH - Length of AT command request resp
+   The sample periodically checks for pending FOTA jobs.
+   The sample performs the FOTA update when received.
+
+If :kconfig:option:`CONFIG_NRF_CLOUD_COAP_FOTA` is enabled, these options additional are available:
+
+.. _CONFIG_COAP_FOTA_DL_TIMEOUT_MIN:
+
+CONFIG_COAP_FOTA_DL_TIMEOUT_MIN - The time in minutes allotted for a FOTA download to complete
+
+.. _CONFIG_COAP_FOTA_USE_NRF_CLOUD_SETTINGS_AREA:
+
+CONFIG_COAP_FOTA_USE_NRF_CLOUD_SETTINGS_AREA - Use the same settings area as the nRF Cloud FOTA library
+
+.. _CONFIG_COAP_FOTA_SETTINGS_NAME:
+
+CONFIG_COAP_FOTA_SETTINGS_NAME - Settings identifier for the CoAP FOTA sample
+
+.. _CONFIG_COAP_FOTA_SETTINGS_KEY_PENDING_JOB:
+
+CONFIG_COAP_FOTA_SETTINGS_KEY_PENDING_JOB - Settings item key for pending FOTA job info
+
+.. _CONFIG_COAP_FOTA_JOB_CHECK_RATE_MINUTES:
+
+CONFIG_COAP_FOTA_JOB_CHECK_RATE_MINUTES - FPTA job check interval (minutes)
+
+.. _CONFIG_COAP_FOTA_THREAD_STACK_SIZE:
+
+CONFIG_COAP_FOTA_THREAD_STACK_SIZE - CoAP FOTA thread stack size (bytes)
+
+
 .. include:: /libraries/modem/nrf_modem_lib/nrf_modem_lib_trace.rst
    :start-after: modem_lib_sending_traces_UART_start
    :end-before: modem_lib_sending_traces_UART_end
 
-.. _nrf_cloud_mqtt_multi_service_building_and_running:
+.. _nrf_cloud_multi_service_building_and_running:
 
 Building and running
 ********************
 
-.. |sample path| replace:: :file:`samples/cellular/nrf_cloud_mqtt_multi_service`
+.. |sample path| replace:: :file:`samples/cellular/nrf_cloud_multi_service`
 
 .. include:: /includes/build_and_run_ns.txt
 
-.. _nrf_cloud_mqtt_multi_service_building_wifi:
+.. _nrf_cloud_multi_service_coap_provisioning:
+
+Provisioning a CoAP device on nRF Cloud
+***************************************
+
+For more information, see `nRF Cloud Utilities documentation`_.
+
+Only do this step once.
+
+.. code-block:: console
+
+   python3 create_ca_cert.py -c US -st OR -l Portland -o "My Company" -ou "RD" -cn example.com -e admin@example.com -f CA
+   Creating self-signed CA certificate...
+   File created: CA0x3bc7f3b014a8ad492999c594f08bbc2fcffc5fd1_ca.pem
+   File created: CA0x3bc7f3b014a8ad492999c594f08bbc2fcffc5fd1_prv.pem
+   File created: CA0x3bc7f3b014a8ad492999c594f08bbc2fcffc5fd1_pub.pem
+
+Do the following step for each device.
+Execute in the same folder as the step above.
+
+.. note::
+   The options used below assume that the sample is built with the default :kconfig:option:`CONFIG_NRF_CLOUD_CLIENT_ID_SRC_IMEI` option enabled and :kconfig:option:`CONFIG_NRF_CLOUD_CLIENT_ID_PREFIX` option set to ``nrf-``.
+   See :ref:`configuration_device_id` to use other device ID formats.
+
+.. code-block:: console
+
+   python3 device_credentials_installer.py --ca CA0x3bc7f3b014a8ad492999c594f08bbc2fcffc5fd1_ca.pem \
+   --ca_key CA0x3bc7f3b014a8ad492999c594f08bbc2fcffc5fd1_prv.pem --id_str nrf- --id_imei -s -d --coap
+   ...
+   <- OK
+   Saving provisioning endpoint CSV file provision.csv...
+   Provisioning CSV file saved
```

Comment:
```suggestion
.. code-block:: console

   python3 device_credentials_installer.py --ca <certificate_serial>_ca.pem \
   --ca_key <certificate_serial>_prv.pem --id_str nrf- --id_imei -s -d --coap
   ...
   <- OK
   Saving provisioning endpoint CSV file provision.csv...
   Provisioning CSV file saved

Where ``<certificate_serial>`` is your certificate's serial number in hex.
```
The console output for the command isn't really necessary since we're only interested in the command.